app/ilmoweb/templates/update_group.html

```diff
@@ -32,14 +32,26 @@ <h1>{{course.name}}</h1><br>
     <div class="row justify-content-center">
       <label for="startTime" class="col-sm-1 col-form-label">Aloitusaika</label>
       <div class="col-3">
-        <input id="startTime" type="time" value="08:00" name="start_time" data-testid="start_time" data-input>
+        <input id="startTime" type="time" 
+        {% if lab_group.start_time.hour < 10 %}
+          value="0{{lab_group.start_time.hour}}:00"
+        {% else %} 
+          value="{{lab_group.start_time.hour}}:00"
+        {% endif %}
+        name="start_time" data-testid="start_time" data-input>
       </div>
     </div>
     <br>
     <div class="row justify-content-center">
       <label for="endTime" class="col-sm-1 col-form-label">Lopetusaika</label>
       <div class="col-3">
-        <input id="endTime" type="time" value="12:00" name="end_time" data-testid="end_time" data-input>
+        <input id="endTime" type="time"
+        {% if lab_group.end_time.hour < 10 %}
```

Comment:
I think we should not hard code the minutes to be 00 in case someone wants to start the group 08:15 etc